docs/pages/validator-management/vc-configuration.md

```diff
@@ -88,11 +88,29 @@ Configure your validator client's fee recipient address by using the `--suggeste
 
 You may choose to use the `--strictFeeRecipientCheck` flag to enable a strict check of the fee recipient address with the one returned by the beacon node for added reassurance.
 
+### Configure your builder selection and/or builder boost factor
+
+If you are running a Lodestar beacon with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) for the beacon node to propose. 
+
+With produceBlockV3 (enabled automatically after the Deneb hard fork), the `builder.boostFactor` is a percentage multiplier the block producing beacon node must apply to boost (>100) or dampen (<100) builder block value for selection against execution block. The multiplier is ignored if `--builder.selection` is set to anything other than `maxprofit`. Even though this is set on the validator client, the calculation is applied on the beacon node itself. For more information, see the [produceBlockV3 Beacon API](https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/produceBlockV3).
+
+With Lodestar's `--builder.selection` options, you can select:
+- `maxprofit`: Default setting for Lodestar set at `builder.boostFactor=100`. This default setting will always choose the more profitable block. Using this option, you may customize your `builder.boostFactor` to your preference. Examples of its usage are below.
+- `executiononly`: Validator will not trigger builder block production even if builder relays are configured on the beacon. This will always select the local execution block.
+- `builderalways`: An alias of `builder.boostFactor=18446744073709551615`, which will select the builder block, unless the builder block fails to produce. The builder block may fail to produce if it's not available, not timely or there is an indication of censorship via `shouldOverrideBuilder` from the execution payload response.
+- `builderonly`: Generally used for distributed validators (DVs). No execution block production will be triggered. Therefore, if a builder block is not produced, the API will fail and *no block will be produced*.
+
+#### Examples using custom `builder.boostFactor` 
+Example 1: Setting a `builder.boostFactor` of `0` is the same as signaling `builder.selection executiononly` where the validator will always select the local execution block if available.
```

Comment:
This is correct with the exception that `executiononly` will reject the block on the validator side if it is not a execution block. It is a very strict "fuck builders" setting, while `builder.boostFactor=0` will produce builder block if local block production failed for some reason.

see code snippet
https://github.com/ChainSafe/lodestar/blob/4996f4d5218994fbbd418be1340a3e816d01eac9/packages/validator/src/services/block.ts#L288-L297